server/controllers/staticController.test.ts

```diff
@@ -31,4 +31,13 @@ describe('StaticController', () => {
       expect(response.render).toHaveBeenCalledWith('static/privacy-notice')
     })
   })
+
+  describe('cookies policy page', () => {
```

Comment:
Is this related to another ticket?

Reply:
yes - I've put in the commit message and PR description, just doing some tidying up to add unit test coverage before starting